docs/visibility-on-elasticsearch.md

```diff
@@ -75,6 +75,25 @@ This part is used to config advanced visibility store to ElasticSearch.
  - `url` is for Cadence to discover ES 
  - `indices/visibility` is ElasticSearch index name for the deployment.  
 
+Optional TLS Support can be enabled by setting the TLS config as follows:
```

Comment:
Thanks for adding documentation. Can you also open a PR in cadence-docs to add the same docs in https://cadenceworkflow.io/docs/concepts/search-workflows/ ?

Reply:
Created: https://github.com/uber/cadence-docs/pull/40